.github/actions/format/action.yml

```diff
@@ -8,4 +8,4 @@ runs:
       run: black --check ./voluncheer
     - name: Lint Python with Flake8
       shell: bash
-      run: flake8 --filename=*.py --exclude=test_*,*_initial.py --max-line-length 100 --ignore=F401 ./voluncheer
+      run: flake8 --filename=*.py --exclude=*_initial.py --max-line-length 100 --ignore=F401 ./voluncheer
```

Comment:
Can we please migrate some, if not all, of this to a `.flake8` config file in the root of the repo? We want the experience to be consistent between CI and local development.

Reply:
I thought you don't want test_* files in CI? I removed the .flake8 because it doesn't seem useful.

Reply:
Or do you want me to add a .flake8 and push?